interfaces/silas_ui.tcl

```diff
@@ -36,6 +36,29 @@ proc iconik_steam_timeout {slot} {
 	return [dict get $::iconik_settings(steam_profiles) $slot timeout]
 }
 
+set ::active_button_indicator {___________________________}
+
+proc is_coffee_chosen { slot } {
+	if {$::settings(profile_title) == [iconik_profile_title $slot]} {
+		return $::active_button_indicator
+	} else {
+		return {}
+	}
+}
+
+proc is_steam_chosen { slot } {
+	if {$::settings(steam_timeout) == [iconik_steam_timeout $slot]} {
+		return $::active_button_indicator
+	} else {
+		return {}
+	}
+}
+
+proc create_active_marker { x1 y1 x2 y2 textvariable } {
```

Comment:
Can go into framework.tcl
pretty handy 🥉

Reply:
> If Silas_UI is sourcing the default settings screen, why create settings_screen.tcl?
> 
> source "[skin_directory]/interfaces/default_settings_screen.tcl" should give you what you need

Sorry not sure about this remark. Isn't that what I do in Line 117. If I don't do it, settings screen does not work. But maybe it could be done simpler?!

Reply:
> Can go into framework.tcl
> pretty handy 🥉

Yes sure. Same applies to create_triple_button. Should I move both? I didn't add the procs there since I though you would prefer that I don't touch anything outside of the new file.

Reply:
Yes, please move both.
Yes you are sourcing the default settings screen, so why do you create a new settingsscreen file?

Reply:
What do you mean by "create a new settingsscreen file"? If I did so, it was not on purpose. But I still don't see where I do this?!